frontend/src/container/QueryBuilder/components/ToolbarActions/tests/ToolbarActions.test.tsx

```diff
@@ -8,7 +8,7 @@ import RightToolbarActions from '../RightToolbarActions';
 describe('ToolbarActions', () => {
 	it('LeftToolbarActions - renders correctly with default props', async () => {
 		const handleChangeSelectedView = jest.fn();
-		const handleToggleShowHistogram = jest.fn();
+		const handleToggleShowFrequencyChart = jest.fn();
```

Comment:
add a test here to check for the text, should be a small addition

Reply:
adding

Reply:
Done